Svc/GenericHub/GenericHub.fpp

```diff
@@ -3,6 +3,22 @@ module Svc {
   @ A generic hub component
   passive component GenericHub {
 
+    # ----------------------------------------------------------------------
+    # Mimic Ports
+    # ----------------------------------------------------------------------
+
+    @ Telemetry input port for mimicking an event processor
+    guarded input port TlmRecv: Fw.Tlm
```

Comment:
I guess I never looked at any of these closely - why are any of the input ports guarded? There's no private data being shared.

Reply:
Made them `sync` as there is no member data.  I think these were originally `guarded` to force an order between accesses and the outgoing data, but that isn't a good argument.